internal/ethapi/api.go

```diff
@@ -1449,19 +1449,29 @@ func newRPCPendingTransaction(tx *types.Transaction, current *types.Header, conf
 func newRPCTransactionFromBlockIndex(b *types.Block, index uint64, config *params.ChainConfig, db ethdb.Database) *RPCTransaction {
 	txs := b.Transactions()
 
+	var txHash common.Hash
+
 	borReceipt := rawdb.ReadBorReceipt(db, b.Hash(), b.NumberU64())
 	if borReceipt != nil {
-		tx, _, _, _ := rawdb.ReadBorTransaction(db, borReceipt.TxHash)
-
-		if tx != nil {
-			txs = append(txs, tx)
+		txHash = types.GetDerivedBorTxHash(types.BorReceiptKey(b.Number().Uint64(), b.Hash()))
```

Comment:
txHash we are getting from here and txHash from borReceipt.TxHash,  are these two same?